docs/source/quick_reference.rst

```diff
@@ -12,20 +12,23 @@ Supported Datasets and Annotations
 
 This table is provided as a guide for users to select appropriate datasets. The
 list of annotations omits some metadata for brevity, and we document the dataset's
-primary annotations only.
+primary annotations only. To access comprehensive details and API documentation for each dataset, please consult the section :ref:`dataset loaders <Datasets API>` within the documentation.
 
 "Downloadable" possible values:
 
 * ✅ : Freely downloadable
 
-* 🔑 : Available upon request
```

Comment:
Also, why removing this?

Reply:
We had a discussion on this and realized that, as there are no datasets or there are rarely going to be any in the future, we decided to remove this.

Reply:
Ok, but for now all datasets are freely downloadable, so `📺 : Youtube Links only` and `❌ : Not available` are not used either. I assume that we expect some datasets to use them in the future? Seems odd to me to remove the `available upon request` but not remove the others. But it's a minor thing, of course. Not very relevant. We can always add a note to the contributing guidelines if necessary.